batcher/aligned/src/main.rs

```diff
@@ -195,7 +212,7 @@ pub struct GetUserBalanceArgs {
     eth_rpc_url: String,
     #[arg(
         name = "The user's Ethereum address",
-        long = "user_addr",
+        long = "user_address",
```

Comment:
Do not change parameters names not related to the pr, since it could break compatibility